config/locales/account_reset/en.yml

```diff
@@ -29,7 +29,7 @@ en:
       title: Deleting your account should be your last resort
     pending:
       cancel_request: Cancel request
-      cancelled: We have cancelled your request to delete your account.
+      cancelled: We have canceled your request to delete your account.
```

Comment:
might be good to rename the key (and at its callsites as well)

Reply:
I am tracking this ticket in https://docs.google.com/document/d/1w6WJbvEI_pWoGmqwLDRlBO3AF7nirehcVDw5ywvG5P8/edit (as I don't have Jira access). In my weakly held opinion, 2 - 5 should be taken up in a separate PR/commit.